src/NServiceBus.Persistence.AzureStorage/Config/AzureSagaPersisterConfig.cs

```diff
@@ -1,52 +0,0 @@
-namespace NServiceBus.Config
-{
-    using System.Configuration;
-
-    /// <summary>
-    /// Config section for the Azure Saga Persister
-    /// </summary>
-    public class AzureSagaPersisterConfig:ConfigurationSection
```

Comment:
I should have verified but I assumed obsoleting a config section wouldn't generate a build error.